### PR TITLE
Rename WGE to WildCam Lab & change background image

### PR DESCRIPTION
**Pull request by shaunanoordin**
- All references to the unofficial project title, Wildcam Gorongosa Education (WGE), have been changed to the official **WildCam Lab**. This follows feedback from HHMI.
  - Website description changed to: "Research & education lab for the WildCam Gorongosa project.", pending approval from Ali and/or HHMI.
- The background image has been changed, following feedback regarding copyright/licence. We now have two options:
  wildcam-new-background-lake
  wildcam-new-background-greenery
  Choose wisely!

Status: **Pending Internal Feedback**
Actions: Project owners to choose between two background picture options; once confirmed, this PR will be updated with the selected picture and the other, unused, image files will be removed.

### CODE REVIEW

**Comment by shaunanoordin** [comment]
The people have spoken! We're going with the Lake Urema background image. @simoneduca , @eatyourgreens , please let me know if there are any issues.

Status: **Ready to merge**

**Comment by simoneduca** [comment]
LGTM

File: src/index.tpl.html [hunk 1]
```diff
@@ -3,8 +3,8 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <title>Wildcam Gorongosa Education</title>
-    <meta name="description" content="Education portal for the WildCam Gorongosa project.">
+    <title>WildCam Lab</title>
+    <meta name="description" content="Research & education lab for the WildCam Gorongosa project.">
```
Comment (eatyourgreens) [inline src/index.tpl.html:7]: Very minor point: should `&` be encoded as `&amp;` here?

**Comment by eatyourgreens** [comment]
:+1:

File: src/index.tpl.html (hunk 1 above)
Comment (simoneduca) [inline src/index.tpl.html:7]: That's a good point. This is still a placeholder anyway.

**Pull request merged**